be/src/olap/storage_engine.h

```diff
@@ -286,6 +289,9 @@ class StorageEngine {
     Status _handle_seg_compaction(BetaRowsetWriter* writer,
                                   SegCompactionCandidatesSharedPtr segments);
 
+    Status _handle_index_change(IndexBuilderSharedPtr index_builder);
+    bool _tablet_busy(TabletSharedPtr tablet);
```

Comment:
```suggestion
    bool _tablet_busy(Tablet* tablet);
```
Copying `shared_ptr` has cacheline synchronization overhead, but no need to copy `shared_ptr` here.